Ghidra/Framework/SoftwareModeling/src/main/java/ghidra/app/plugin/assembler/sleigh/SleighAssembler.java

```diff
@@ -24,6 +24,7 @@
 import ghidra.app.plugin.assembler.sleigh.sem.*;
 import ghidra.app.plugin.assembler.sleigh.symbol.AssemblyNumericSymbols;
 import ghidra.app.plugin.assembler.sleigh.util.DbgTimer;
+import ghidra.app.plugin.querylanguage.lexer.ast.InstructionNode;
```

Comment:
I'd rather not see a dependency from assembler to your query language. We should try to design an interface instead.